src/app/cookie-policy/cookie-policy.component.html

```diff
@@ -1,11 +1,12 @@
 <div class="fit-to-page">
   <div class="page-title-wrapper">
-    <h2 class="page-title" data-e2e="cookie-policy">Cookie Policy</h2>
+    <h2 class="page-title" data-e2e="cookie-policy">Cookies Policy</h2>
   </div>
 
   <p>
     This cookies policy applies to the Europeana website
```

Comment:
instead also `Europeana` should be here `metis statistics` or `statistics dashboard` website?
![afbeelding](https://github.com/europeana/statistics-dashboard/assets/19560266/8d5b096c-eca3-40c3-84ad-a70c5a424005)

Reply:
As discussed for the sandbox, the final text in production will look different.